pkg/interfaces/contracts/vault/IBasePool.sol

```diff
@@ -21,14 +21,23 @@ interface IBasePool is ISwapFeePercentageBounds, IUnbalancedLiquidityInvariantRa
     /**
      * @notice Computes the pool's invariant.
      * @dev This function computes the invariant based on current balances (and potentially other pool state).
+     * The given rounding direction must be respected for the Vault to round in its favor when calling this function.
```

Comment:
```suggestion
     * The given rounding direction must be respected for the Vault to round in pool's favor when calling this function.
```

I'd put in pool's favor, since we need the developers of the pool to understand why the rounding is important.